packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -6,82 +6,26 @@ section: developer-resources
-
-`--pf-v6-global--BackgroundColor--400` was added as a background color for hover and interaction states of items that are placed on top of a dark background, such as dropdown menu items, text inputs, text areas, and menu toggles.
-
-`--pf-v6-global--BorderColor--400` was added as a bottom border color for control elements.
-
-**Status colors**
-
-`--pf-v6-global--primary-color--300` was added as a new primary background color to meet contrast requirements for elements that have text on top, such as primary buttons and unread badges.
-
-**Component style overrides**
-
-Where there are dark theme style overrides needed for specific components, you can find them alongside the components stylesheet in a folder with the structure `themes/dark/[component].scss`. For example, the button component's dark theme overrides are in [components/Button/themes/dark/button.scss](https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Button/themes/dark/button.scss)
+[Our token resources](https://github.com/patternfly/patternfly/tree/v6/src/patternfly/base/tokens) include styles for both light and dark themes. When dark theme is enabled, your product will automatically pull dark theme tokens in order to adapt visual styles appropriately.
 
 ## Best practices
 
-To make it easier to support light and dark theming in your application, we recommend considering these best practices during design and development:
-
-* Using PatternFly components without customizations will make it easier to implement dark theme in your application. Most of the problems encountered have been around the use of custom styles, overrides to PatternFly styles, and non-PatternFly components. To ensure consistent styling and behavior when switching themes, try to use PatternFly components as they are.
-
+To make it easier to support light and dark theming in your application, we recommend following these best practices during design and development:
 
-* When making overrides, utilize the global and component CSS variable system to define the override. For example, to override a primary button’s background color, declare `.pf-v6-c-button { --pf-v6-c-button--m-primary--BackgroundColor: [color]; }` instead of `.pf-v6-c-button.pf-m-primary { background-color: [color]; }`.
+* **Use PatternFly components without customizations.** Most problems with dark theming are due to the use of custom styles, overrides to PatternFly styles, and non-PatternFly components. To ensure consistent styling and behavior when switching themes, try to use PatternFly components as they are.
 
+* **Use tokens to define any overrides.** 
+    * For example, to override a primary button’s background color, declare `.pf-[version]-c-button { --pf-[version]-c-button--m-primary--BackgroundColor: [color]; }` instead of `.pf-[version]-c-button.pf-m-primary { background-color: [color]; }`.
```

Comment:
It might be good to say `[color token]` instead of `[color]` - it's technically fine but at first read it seems to say that assigning it to a hex code would be fine as long as you use the component variable.